cpg-core/src/test/kotlin/de/fraunhofer/aisec/cpg/passes/CallResolverTest.kt

```diff
@@ -86,13 +83,13 @@ class CallResolverTest : BaseTest() {
      *
      * @param result
      */
-    private fun ensureInvocationOfMethodsInFunction(result: List<TranslationUnitDeclaration>) {
-        assertEquals(1, result.size)
-        val tu = result[0]
+    private fun ensureInvocationOfMethodsInFunction(result: TranslationResult) {
+        assertEquals(1, result.translationUnits.size)
+        val tu = result.translationUnits[0]
         for (declaration in tu.declarations) {
             assertNotEquals("invoke", declaration.name)
         }
-        val callExpressions = flattenListIsInstance<CallExpression>(result)
+        val callExpressions = result.allChildren<CallExpression>()
```

Comment:
Can't we automatically replace `result.allChildren<CallExpression>()` with `result.calls`?

Reply:
Probably :)